package.json

```diff
@@ -57,6 +57,7 @@
     "nprogress": "0.2.0",
     "nuka-carousel": "4.4.4",
     "payment": "2.3.0",
+    "polished": "^2.3.1",
```

Comment:
Use fixed version.

Reply:
Good catch, I should take the habit of installing with `--save-exact`